### PR TITLE
Schema bump

### LINKED ISSUE #2266: general: bump inspire-schemas to version ~32.0

### PR DESCRIPTION
**Pull request by Glignos**
## Related Issue
Closes #2266 

## Checklist:
- [ ] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [ ] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [ ] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by david-caro** [comment]
You did not actually change the setup.py right? (to pin the schemas to the new version)

File: inspirehep/dojson/experiments/rules.py [hunk 1]
```diff
@@ -48,56 +48,56 @@ def date_started(self, key, value):
             self['date_started'] = val.get('s')
         if val.get('t'):
             self['date_completed'] = val.get('t')
+        if val.get('c'):
+            self['date_cancelled'] = value.get('c')
```
Comment (david-caro) [inline inspirehep/dojson/experiments/rules.py]: We were actually  missing this xd (it's not a change introduced by the bump, but a previous missing field)

Comment (Glignos) [inline inspirehep/dojson/experiments/rules.py]: Ah well :coffee: 
Also I changed the setup.py locally but I didn't commit it it in the PR

File: tests/unit/dojson/test_dojson_institutions.py [hunk 2]
```diff
@@ -758,7 +758,7 @@ def test_related_institutes_from__510_a_w_0():
     assert expected == result['related_institutes']
 
 
-def test_related_institutes_from__double_510_a_w_0():
+def test_related_institutes_from__double_510_w_0():
```
Comment (jacquerie) [inline tests/unit/dojson/test_dojson_institutions.py]: Why are you changing the test names? The test names denote the snippet that you're using. If the snippet doesn't change (and there's usually no reason to change the snippet), the test name doesn't change either!

File: tests/unit/dojson/test_dojson_experiments.py [hunk 3]
```diff
@@ -29,210 +30,212 @@
...
-        '</datafield>'
+        '<record>'
+        '  <datafield tag="046" ind1=" " ind2=" ">'
+        '    <subfield code="q">2014</subfield>'
+        '    <subfield code="r">2015</subfield>'
+        '    <subfield code="s">1995</subfield>'
+        '    <subfield code="t">2008</subfield>'
+        '    <subfield code="c">2016</subfield>'
+        '  </datafield>'
+        '</record>'
```
Comment (jacquerie) [inline tests/unit/dojson/test_dojson_experiments.py]: This test is way too optimistic: you'll never find data this clean in INSPIRE legacy. Look at what's actually in the records!

File: tests/unit/dojson/test_dojson_experiments.py [hunk 4]
```diff
@@ -29,210 +30,212 @@
...
+    subschema_inst = schema['properties']['institution']
 
     snippet = (
         '<record>'
         '  <datafield tag="119" ind1=" " ind2=" ">'
         '    <subfield code="a">CERN-ALPHA</subfield>'
         '    <subfield code="u">CERN</subfield>'
-        '    <subfield code="z">902725</subfield>'
+        '    <subfield code="c">NA61</subfield>'
+        '    <subfield code="b">LHC</subfield>'
```
Comment (jacquerie) [inline tests/unit/dojson/test_dojson_experiments.py]: Never modify previous tests: they are there to guarantee that you're not breaking earlier behavior.

File: inspirehep/dojson/experiments/rules.py [hunk 5]
```diff
@@ -30,6 +30,7 @@
 from dojson.errors import IgnoreKey
 
 from inspirehep.utils.helpers import force_list
+from inspire_schemas.api import LiteratureBuilder
```
Comment (jacquerie) [inline inspirehep/dojson/experiments/rules.py]: Somehow I doubt that you're going to use the `LiteratureBuilder` while creating experiment records...

**Comment by jacquerie** [comment]
Since I will need #2267 to be done for #2227, and #2267 depends on #2266, I will take over this PR.

File: inspirehep/dojson/experiments/rules.py [hunk 6]
```diff
@@ -48,59 +48,121 @@ def date_started(self, key, value):
...
+        'Reactor': '3.2',
+        'Accelerator': '3.1',
+        'Modified gravity and space-time': '6.2',
+        'Muon beams': '7.2',
+        'Accelerator Test Facility Experiments': '7',
+        'ultra-short-baseline': '3.2.1',
+        'Neutrinoless double beta decay': '3.4'
+        }
+
+    return experiments_list[value.get('a')]
```
Comment (jacquerie) [inline inspirehep/dojson/experiments/rules.py]: FYI: this wasn't correct, see https://github.com/inspirehep/inspire/issues/301 for what should have happened here.

**Pull request closed without merging**